web/modules/custom/ai_screening_project_track/src/Entity/ProjectTrack.php

```diff
@@ -293,8 +293,10 @@ public function getDimensions(): array {
   public function getSummedValues(): ?array {
     $values = [];
     $config = $this->getConfiguration();
-    foreach ($config['bubbleChartReportResult']['sums'] as $key => $sum) {
-      $values[$key] = $sum['sum'];
+    if (isset($config['bubbleChartReportResult'])) {
+      foreach ($config['bubbleChartReportResult']['sums'] as $key => $sum) {
```

Comment:
Check that the value you're actually using exists:

```suggestion
    if (isset($config['bubbleChartReportResult']['sums'])) {
      foreach ($config['bubbleChartReportResult']['sums'] as $key => $sum) {
```

To be on the safe side, you could also check that the values is an array, but we're allowed to make some assumptions about the data we're juggling around.